fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/PaloAuth.java

```diff
@@ -584,9 +584,13 @@ private void createUserInternal(UserIdentity userIdent, String roleName, byte[]
                     throw new DdlException("Role: " + roleName + " does not exist");
                 }
             }
-            
+
             // 2. check if user already exist
             if (userPrivTable.doesUserExist(userIdent)) {
+                if (ignoreIfExists) {
+                    LOG.debug("user exists, ignored to create user: {}, is replay: {}", userIdent, isReplay);
```

Comment:
@morningman Maybe this should be `LOG.info`, instead of changing all other `LOG.info` to `LOG.debug`. 

If we use `IF NOT EXISTS` but the user exists, we should be informed instead of the message being swallowd. 

The surrounding code sugguests using `LOG.info` (L623).